src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -51,15 +51,14 @@ export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$, breadcrumbsAppendEx
     ),
   }));
 
-  if (breadcrumbsAppendExtension) {
+  if (breadcrumbsAppendExtension && crumbs[crumbs.length - 1]) {
```

Comment:
NIT: isn't `bool(crumbs[crumbs.length - 1])` just `bool(crumbs.length)` but harder to read 😅 ?